### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by andgarzonmal**
### FOR MILESTONE 2

- [x]  Add a column species of type string to your animals table. Modify your schema.sql file.

image

- [x]  Insert new data 

image

- [x]  Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

image

image

- [x]  Inside a transaction:
•	Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
•	Update the animals table by setting the species column to pokemon for all animals that don't have species already set.

image

- [x]  took a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.

image

- [x]  After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)

image

- [x]  Inside a transaction:
    o	Delete all animals born after Jan 1st, 2022.
    o	Create a savepoint for the transaction.
    o	Update all animals' weight to be their weight multiplied by -1.
    o	Rollback to the savepoint
    o	Update all animals' weights that are negative to be their weight multiplied by -1.
    o	Commit transaction.

image

- [x]  How many animals are there?

image

- [x]  How many animals have never tried to escape?

image

- [x]  What is the average weight of animals?

image

- [x]  Who escapes the most, neutered or not neutered animals?

image

- [x]  What is the minimum and maximum weight of each type of animal?

image

- [x]  What is the average number of escape attempts per animal type of those born between 1990 and 2000?

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -41,3 +41,63 @@ SELECT *
...
+
+-- MILESTONE 2
+-- Inside a transaction update the animals table by setting the species column to unspecified. 
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+    UPDATE animals SET species = 'unespecified';
+    ROLLBACK;
+COMMIT;
+
```
Comment (thecodechaser) [inline queries.sql:64]: - [ ] Your project is missing the below query, Please make sure to add this transaction in the queries.
image

File: queries.sql [hunk 2]
```diff
@@ -41,3 +41,63 @@ SELECT *
...
     WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- MILESTONE 2
+-- Inside a transaction update the animals table by setting the species column to unspecified. 
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+    UPDATE animals SET species = 'unespecified';
+    ROLLBACK;
+COMMIT;
```
Comment (thecodechaser) [inline queries.sql]: - [ ] On this transaction, Please verify that the change was made. Hint: `SELECT species from animals;`.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @andgarzonmal👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @andgarzonmal, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**